src/main/java/org/syftkog/web/test/framework/BrowserVersionPlatform.java

```diff
@@ -0,0 +1,131 @@
+package org.syftkog.web.test.framework;
+
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+
+/**
+ *
+ * @author BenjaminLimb
+ */
+public enum BrowserVersionPlatform {
+
+  /**
+   *
+   */
+  WIN7IE8("ie8", "internet explorer", "8", "Window 7"),
+
+  /**
+   *
+   */
+  WIN7IE9("ie9", "internet explorer", "9", "Window 7"),
+
+  /**
+   *
+   */
+  WIN7IE10("ie10", "internet explorer", "10", "Window 7"),
+
+  /**
+   *
+   */
+  WIN7IE11("ie11", "internet explorer", "8", "Window 7"),
+
+  /**
+   *
+   */
+  WIN7FF("ff", "firefox", "any", "Windows 7"),
+
+  /**
+   *
+   */
+  WIN7CHOME("chrome", "chrome", "any", "Windows 7"),
+
+  /**
+   *
+   */
+  HTMLUNIT("htmlunit", "htmlunit", "any", "any"),
+
+  /**
+   *
+   */
+  PHANTOMJS("phantomjs", "phantomjs", "any", "any"),
+
+  /**
+   *
+   */
+  MOCK("mock", "mock", "any", "any");
+
+    private String key;

```

Comment:
Although you clearly don't have any setter methods for these fields you might want to consider marking these as `final` just the same as this is an enum and it would be difficult to troubleshoot an issue if these were ever to be changed after initialization.